lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -549,7 +552,53 @@ def upload_csv_to_report_store(rows, csv_name, course_id, timestamp):
     )
 
 
-def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input, action_name):
+def verification_status_for_user(user, course_id, user_enrollment_mode):
+    """
+    Returns the verification status.
+    """
+    if user_enrollment_mode not in CourseMode.VERIFIED_MODES:
+        return 'ID Verified'
+
+    user_is_verified = SoftwareSecurePhotoVerification.user_is_verified(user)
+
+    if not user_is_verified:
+        return 'Not ID Verified'
+    else:
+        user_is_re_verified = SoftwareSecurePhotoVerification.user_is_reverified_for_all(course_id, user)
+        if not user_is_re_verified:
+            return 'ID Verification Expired'
+        else:
+            return 'ID Verified'
+
+
+def certificate_info_for_user(user, course_id, whitelist_user_ids, grade):
```

Comment:
Can you also move this to `certificates/models.py`?

Also instead of passing in the list of `whitelist_user_ids`, it would make more sense to have `user_is_whitelisted=None` in the parameters. And then do before line 578:

```
if user_is_whitelisted is None:
  user_is_whitelisted = <query database >
```